pkg/rbac/userData.go

```diff
@@ -258,6 +258,9 @@ func (user *UserDataCache) getNamespacedResources(cache *Cache, ctx context.Cont
 		} else {
 			klog.V(9).Infof("SelfSubjectRulesReviews Kube API result for ns:%s : %v\n", ns, prettyPrint(result.Status))
 		}
+
+		// Name the loop - to break out of the loop if user is kubeadmin (have access to everything)
```

Comment:
The user doesn't have to be 'kubeadmin', other users could have similar `*` access.
```suggestion
		// Name the loop - to break out of the loop if user has access to everything (*)
```

Reply:
Updated